src/mockData/GitcoinCurriculumData.jsx

```diff
@@ -1,87 +1,95 @@
-    contentType: 'video',
+    contentType: "video",
     time: 1000,
-    link: 'https://www.youtube.com/watch?v=KpXPym_m_wA',
+    link: "https://www.youtube.com/watch?v=KpXPym_m_wA",
     tokens: 5,
   },
   {
-    id: 5,
-    title: 'Select your first task and project',
+    id: 4,
+    title: "Select your first task and project",
     description:
-      'Select your first task and project to get started as a member of the DAO',
-    contentType: 'image',
+      "Select your first task and project to get started as a member of the DAO",
+    contentType: "image",
     time: 300,
-    link: '/GitcoinDAO-screenshot.png',
+    link: "/GitcoinDAO-screenshot.png",
     tokens: 20,
   },
   {
-    id: 6,
-    title: 'Register for dRewards',
+    id: 5,
+    title: "Register for dRewards",
     description:
-      'Register for dRewards and get a free token to start working on your first task and project',
-    contentType: 'image',
-    time: '--',
-    link: '/logos/GitcoinDAO.png',
+      "Register for dRewards and get a free token to start working on your first task and project",
+    contentType: "image",
+    time: "--",
+    link: "/logos/GitcoinDAO.png",
     tokens: 10,
   },
   {
-    id: 7,
-    title: 'Receive your first dReward',
+    id: 6,
+    title: "Receive your first dReward",
     description:
-      'Now you are an official contributor with a verifiable onchain history!',
-    contentType: 'image',
-    time: '--',
-    link: '/logos/GitcoinDAO.png',
+      "Now you are an official contributor with a verifiable onchain history!",
+    contentType: "image",
+    time: "--",
+    link: "/logos/GitcoinDAO.png",
     tokens: 10,
   },
   {
-    id: 8,
+    id: 7,
     title:
-      'Wrap up: You are comfortable with streams, squads, and dRewards and are ready to rock and roll.',
+      "Wrap up: You are comfortable with streams, squads, and dRewards and are ready to rock and roll.",
     description:
-      'Woot woot! You are ready to rock and roll! Looking forward to seeing you on the DAO side! Remember to high five those you have learned alongside with.',
-    contentType: 'video',
+      "Woot woot! You are ready to rock and roll! Looking forward to seeing you on the DAO side! Remember to high five those you have learned alongside with.",
+    contentType: "video",
     time: 300,
-    link: 'https://youtu.be/OnoNITE-CLc?t=64',
+    link: "https://youtu.be/OnoNITE-CLc?t=64",
     tokens: 20,
   },
 ];
 
-export default GitcoinCurriculumData;
+const GitcoinCurriculumDataOutcomes = [
+  "Have an understanding of the values & mission of GitcoinDAO",
+  "Have an understanding of how they can earn with GitcoinDAO",
+  "Have an understanding of how to find opportunities and apply to them",
+  "Have key contacts to go to for help & other questions",
+  "Have proper access & permissions to get started",
+];
+
+export { GitcoinCurriculumDataOutcomes, GitcoinCurriculumData };
```

Comment:
Any plan to insert this data into the database and read from there?

Reply:
Yes. It is currently setup that way on my latest branch